internal/kafka/internal/clusters/ocm_provider.go

```diff
@@ -442,7 +450,7 @@ func (o *OCMProvider) CreateMachinePool(request *types.MachinePoolRequest) (*typ
 	return request, err
 }
 
-// GetQuotaCosts returns a list of ocm resource quota cost information for the authenticated user
+// GetQuotaCosts returns a list of data plane related ocm resource quota cost information for the authenticated user
 // Returns a nil slice when no resource quota is assigned to the user
 func (o *OCMProvider) GetQuotaCosts() ([]types.QuotaCost, error) {
```

Comment:
Is there some reason we don't include rhosak related quotacosts in here?
In any case we should document the behavior of this as the method name is very generic but we return very specific results

Reply:
The rhosak related quota costs are for creating kafkas and we don't need to care about that since those are only used for testing purposes.

Would changing the method name here be better? Maybe change it to `GetClusterResourceQuotaCosts`? Then I can add a comment in here that states this will only return quota related to cluster resources used when provisioning and terraforming a data plane cluster. wdyt?

Reply:
Changing the name seems a good idea.